src/ConferenceList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ConferenceItem from "./ConferenceItem";
+import conferences from "./conferenceDataMock.json";
+
+const ConferenceList = () => {
+  return (
+    <div className="conference list">
```

Comment:
This className property properly works differently to what you expect: You're giving it two classes conference and list. Css classes usually are written in camelCase or with under scores but can't contain whitespaces, so: conferenceList or conference_list

Reply:
Also to add onto this, I'd argue if you need the div at all? You could have the ul as the parent element and give that the className

Reply:
Yeah, I agree

Reply:
Ok we've changed it! :)